docs/foundation/agency-theming.md

```diff
@@ -36,6 +36,18 @@ You’ll find the full list of themeable parts for Pelican 2 in the Pelican 2 Th
 
 {% include 'color-warning.njk' %}
 
+## Background Texture
+
+Themed background texture tiles were included in Pelican by default using SVG inlined in the CSS. You can see an example of it in action in the [Get Started section of the Pelican home page](/). This added considerable weight to the CSS so this has been changed so that these themed textures are optionally loaded. The background colors of these optional CSS files can be themed using CSS variables.
+
+Load these CSS files in the <head> section of your project after Pelican CSS and your theme file. You can get them at the following locations.
```

Comment:
How does one apply the background texture?

Reply:
Added more instructions and an example